pyrobuf/protobuf/templates/proto_pyx.tmpl

```diff
@@ -10,6 +10,12 @@ import json
 from {{ import }}_proto cimport *
 {%- endfor %}
 
+{%- macro message_enum_fields_def(enum) %}
```

Comment:
Suppose you define an enum inside a message like this:

```
message Test {
    enum test_enum {
        TEST_ENUM_FIELD_1 = 1;
    }
}
```

In Google's protobuf, the `Test` class will have a class-level attribute called `TEST_ENUM_FIELD_1` with the integer value 1. This macro replicates that behaviour in pyrobuf.